examples/simple.js

```diff
@@ -1,21 +1,23 @@
-require('../assets/index.less');
-const React = require('react');
-const ReactDOM = require('react-dom');
-const Switch = require('rc-switch');
+import '../assets/index.less';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Switch from 'rc-switch';
 
 function onChange(value) {
   console.log(`switch checked: ${value}`); // eslint-disable-line
 }
 
-class Test extends React.Component {
-  state = {
-    disabled: false,
-  };
-  toggle = () => {
+const Test = React.createClass({
```

Comment:
Why??? We should not use `React.createClass` after React@15.5, just use ES6 class.